src/core/providers/strategies/LedgerProviderStrategy/helpers/signLedgerMessage.ts

```diff
@@ -0,0 +1,21 @@
+import { Message } from '@multiversx/sdk-core/out';
+import { providerLabels } from 'core/providers/types/providerFactory.types';
+import { getLedgerErrorCodes } from './getLedgerErrorCodes';
+import { signMessage } from '../../helpers/signMessage/signMessage';
+
+export async function signLedgerMessage(
+  message: Message,
```

Comment:
I still believe this should take an object as argument, since the below `signMessage` takes the same parameters pretty much and it's an object. We should aim for consistency. But you can leave it as it is if you don't agree.